features/cli.feature

```diff
@@ -1,5 +1,18 @@
 Feature: `wp cli` tasks
 
+  Scenario: Ability to detect a WP-CLI registered command
+    Given an empty directory
+    And save the {SRC_DIR}/VERSION file as {TRUE_VERSION}
+    And a new Phar with version "1.3.0"
```

Comment:
Why do you need to generate a new Phar here?

Reply:
You're correct, there wasn't a need to generate a phar. I replaced it with `wp` @danielbachhuber